pkg/xds/server/v3/reconcile.go

```diff
@@ -173,9 +173,15 @@ func (s *TemplateSnapshotGenerator) GenerateSnapshot(ctx xds_context.Context, pr
 		reconcileLog.Error(err, "failed to generate a snapshot", "proxy", proxy, "template", template)
 		return nil, err
 	}
-	for name, p := range plugins.Plugins().PolicyPlugins() {
-		if err := p.Apply(rs, ctx, proxy); err != nil {
-			return nil, errors.Wrapf(err, "could not apply policy plugin %s", name)
+	policies := plugins.Plugins().PolicyPlugins()
+	for _, policyName := range ctx.ControlPlane.EnabledPolicies {
+		policy, exists := policies[plugins.PluginName(policyName)]
+		if !exists {
+			reconcileLog.Info("there is no pluggable policy, skip", "policyName", policyName)
```

Comment:
I think this likely an error no? Someone references something that doesn't exist